api/IDBKeyRange.json

```diff
@@ -50,12 +57,26 @@
           "safari_ios": {
             "version_added": "8"
           },
-          "samsunginternet_android": {
-            "version_added": true
-          },
-          "webview_android": {
-            "version_added": true
-          }
+          "samsunginternet_android": [
+            {
+              "version_added": "1.5"
+            },
+            {
+              "version_added": "1.5",
+              "version_removed": "7.0",
+              "prefix": "webkit"
```

Comment:
This prefixed support should be on Opera as well. For consistency, can you add it there (my preference) or alternatively skip it here too?